### PR TITLE
Overview content-prior to operator and author content

### PR DESCRIPTION
**Pull request by SteveLasker**
Signed-off-by: Steven Lasker <stevenlasker@hotmail.com>

### CODE REVIEW

**Comment by SteveLasker** [comment]
Consumes #3 & #4 with overview content that links to the previous commits.

File: README.md [hunk 1]
```diff
@@ -19,6 +26,48 @@ By providing an OCI artifact definition, the community can continue to innovate,
...
+
+### Artifacts
+
+Artifacts, like docker and OCI images are well known types of artifacts. Using the same schemas and persistence, additional artifacts can be defined, persisted and served with OCI distribution spec compliant registries.
+
+OCI Artifacts generalizes registries from uniquely storing OCI Images, to generic means storing additional artifact types.
+
+## Defining OCI Artifact Types
+
+[Registries][def-registry], vulnerability scanners and artifact tooling must understand the types of artifacts they support. Registry scanning tools may only support a subset of artifact types, or they may need to apply different scanning methods based on the artifact type.
```
Comment (mikebrow) [inline README.md]: see earlier comment regarding a separate treatise

File: README.md [hunk 2]
```diff
@@ -19,6 +26,48 @@ By providing an OCI artifact definition, the community can continue to innovate,
...
+
+If a security scanning solution were to scan all types, it would fail when it encounters unsupported types, representing false negatives. By differentiating types, a registry scanning solution can ignore unknown types, representing a known state. As new artifact types become [well known][def-well-known-types], scanners can expand the types they offer, providing a more complete known state.
```
Comment (mikebrow) [inline README.md]: same

File: README.md [hunk 3]
```diff
@@ -19,6 +26,48 @@ By providing an OCI artifact definition, the community can continue to innovate,
...
+
+Artifact tooling must also know the types they support. The docker and containerD client know how to instance container images. However, they are not intended to instance Helm Charts or Singularity images. By defining the artifact type, registries can present the type to their users, and tools pulling artifacts from a registry can determine if they can support the specific type before encountering a runtime error.
```
Comment (mikebrow) [inline README.md]: same

File: README.md [hunk 4]
```diff
@@ -19,6 +26,48 @@ By providing an OCI artifact definition, the community can continue to innovate,
...
+
+Artifacts are defined by setting the `manifest.config.mediaType` to a globally unique value. The `config.mediaType` of `application/vnd.oci.image.config.v1+json` is reserved for artifacts intended to be instanced by docker and [containerD][containerd].
```
Comment (mikebrow) [inline README.md]: yes but to much detail for the readme...we should cover this in the spec vs readme..

s/containerD/containerd/

File: definitions-terms.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
+# Definitions and Terms
```
Comment (mikebrow) [inline definitions-terms.md]: wrong PR

File: implementors.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# OCI Artifacts Implementations
```
Comment (mikebrow) [inline implementors.md]: ... wrong pr

File: README.md [hunk 7]
```diff
@@ -2,15 +2,22 @@
...
-By providing an OCI artifact definition, the community can continue to innovate, focusing on new artifact types without having to build yet another storage solution (YASS). 
+By providing an OCI artifact definition, the community can continue to innovate, focusing on new artifact types without having to build yet another storage solution ([YASS][def-yass]).
 
-## Table of Contents
+## OCI Artifact Table of Contents
+
+* [Overview of Registry Content Delivery](#overview-of-registry-content-delivery)
+* [Defining OCI Artifact Types](#defining-oci-artifact-types)
+* [Definitions & Terms](definitions-terms.md)
+* [OCI Artifact Implementations](implementors.md)
```
[the remark this answers is not in the document]
Comment (SteveLasker) [reply README.md]: On the logos, what do you suggest here? Can I reference logos, via a URL to their official logo? Do I have to resort to text :(
Can I ask the maintainers of Helm and Singularity to make a PR, this way it's their contribution? 
Eventually, for the artifactTypes folder, I was planning on asking artifact authors to submit their logo for general consumption, so they are giving rights to others to use when displaying the type within their registry products & services.

[the remark this answers is not in the document]
Comment (mikebrow) [reply README.md]: yeah just reference their logo ..

**Comment by mikebrow** [comment]
@SteveLasker how would you like to resolve these comments.. I could carry if you like

**Comment by SteveLasker** [comment]
Thanks @mikebrow 
this is some cleanup when I was intermixing layered PRs to boot up the artifacts repo. 
If this is truly overview content (i need to look closer) then it’s outdated.
There’s some content for localized identifiers for each artifact we want to preserve and another doc for what registries need to do to enable artifacts. It was about loosening the restrictions on the config.mediaTyoe and layer.mediaType.

**Comment by SteveLasker** [comment]
This PR is outdated with various other PRs.

**Pull request closed without merging**